### PR TITLE
2 product view

### PR DESCRIPTION
**Pull request by JulieBusch**
Hello! So, we have finished the Product model and tests to our satisfaction. We made a branch for our story (Product Catalog View), so we also made changes in some of our routes. We then realized that maybe we should do a pull request on just the model-related stuff we've done so far, but our commits are all intermingled with some route work and some model work and we can't find a way to separate them out into different branches at this point. So, we're submitting this branch so we can get our model stuff added in, but we're going to start a new branch that focuses on backend routes and flesh out the product subrouter and make tests. Then we'll move on to front end branching. Sound good?

### CODE REVIEW

**Comment by emilymalecbrown** [comment]
this all looks great! you can choose one of the two changes below to make -- add the productSeed to your package.json seed script and keep the two seed files
```"seed": "node db/seed.js && node db/productSeed.js")```
or move the seeds into one file and chain the .then()s
```
db.didSync
  .then(() => db.sync({force: true}))
  .then(seedUsers)
  .then(users => console.log(`Seeded ${users.length} users OK`)) 
  .then(seedProducts)
  .then(products => console.log(`Seeded ${products.length} products OK`))
```
talk to your group about which way you prefer! one results in a long seed file the other results in just more files.

File: server/products.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const db = require('APP/db');
+const router = require('express').Router();
+const Product = db.model('products');
+
+router.get('/', function(req, res, next) {
+  // res.send('Hellooooooo');
+  Product.findAll()
```
Comment (emilymalecbrown) [inline server/products.js]: for cleanliness, I'd delete the comment on line 8 -- but as you go on to have more complicated routes, maybe add comments about what the routes are hitting / used for.

**Comment by goldienova** [comment]
Made the changes as requested!  Should be good.  Thanks!

**Comment by emilymalecbrown** [comment]
🏆

**Pull request merged**